app/assets/javascripts/fae/_tables.js

```diff
@@ -267,6 +267,8 @@ Fae.tables = {
 
       Fae.tables.sizeFixedHeader();
     });
+
+    $toggle.trigger('click');
```

Comment:
@aaronransley Did you look into updating classes and CSS first? I would prefer rendering the open-by-default state to begin with via CSS vs rendering them closed and using a JS click to open them. What do you think?